src/tests/example.spec.js

```diff
@@ -24,4 +26,65 @@ test.describe('', () => {
         await shopingCartPage.removeCartItemById(0);
         await expect(shopingCartPage.cartItems).not.toBeAttached();
     });
+
+    test('Perform and verify sorting on the Inventory page', async ({ inventoryPage, loginPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+        await test.step('Sorting products list', async () => {
+            await inventoryPage.sortingBtn.click();
+
+            const sortList = inventoryPage.sortingBtn.locator('option');
+            await expect(sortList).toHaveText([
+                'Name (A to Z)',
+                'Name (Z to A)',
+                'Price (low to high)',
+                'Price (high to low)']);
+
+            await inventoryPage.switchSorting('hilo');
+        });
+        const list = inventoryPage.itemsPrice;
+        const firstItem = await inventoryPage.inventoryItemsName.first().textContent();
+
+        await test.step('Check the order items from High to Low', async () => {
+            if (firstItem === 'Sauce Labs Fleece Jacket') {
+                await inventoryPage.validateChosenProducts(list);
+            }
+        });
+    });
+
+    test('Verification products in Shopping cart', async ({ inventoryPage, shopingCartPage, loginPage }) => {
+        await loginPage.navigate();
+        await loginPage.performLogin('standard_user', 'secret_sauce');
+
+        await inventoryPage.addRandomProducts(6);
```

Comment:
let's randomise amount of products to be added as well

Reply:
create utils function for that